### PR TITLE
Update Under the Hood Intro

### PR DESCRIPTION
**Pull request by Bharat123rox**
## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

Hyperlinks and bullet points are a mix of quoted and unquoted text, this aims to make them uniform across the document

## What is the new behavior?

Move bullet points and hyperlinks into quoted text 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: doc/blog/201808-UnoUnderTheHoodIntro.md [hunk 1]
```diff
@@ -28,15 +28,15 @@ top of [WebAssembly](https://webassembly.org/).
 
 There are a few things that need to be done to get your app running: 
 
--   At compile time, Uno parses XAML files into C\# code, creating the
+-   > At compile time, Uno parses XAML files into C\# code, creating the
```
Comment (XiaotianNetlift) [inline doc/blog/201808-UnoUnderTheHoodIntro.md]: Would be better to remove the quotation from this list.

File: doc/blog/201808-UnoUnderTheHoodIntro.md [hunk 2]
```diff
@@ -50,18 +50,17 @@ we've built this thing. 
 
 -   **Leverage existing tools** 
 
-    -   We stand on the shoulders of  giants. Microsoft's tooling is a
-        > treat to work with: [Edit and
-        > Continue](https://docs.microsoft.com/en-us/visualstudio/debugger/edit-and-continue), [Live
-        > Visual
-        > Tree](https://docs.microsoft.com/en-us/visualstudio/debugger/inspect-xaml-properties-while-debugging),
+    -   > We stand on the shoulders of  giants. Microsoft's tooling is a
```
Comment (XiaotianNetlift) [inline doc/blog/201808-UnoUnderTheHoodIntro.md]: list items' quotation from this list don't need  another level of bullet.

Comment (Bharat123rox) [reply doc/blog/201808-UnoUnderTheHoodIntro.md]: Done

File: doc/blog/201808-UnoUnderTheHoodIntro.md (hunk 1 above)
Comment (Bharat123rox) [reply doc/blog/201808-UnoUnderTheHoodIntro.md]: Done

**Comment by Bharat123rox** [comment]
Gentle ping @XiaotianNetlift

**Pull request merged**